editor/components/post-excerpt/index.js

```diff
@@ -47,4 +49,3 @@ export default connect(
 		},
 	}
 )( PostExcerpt );
```

Comment:
To the above point on `withInstanceId`, you can either update this line to:

```js
)( withInstanceId( PostExcerpt ) );
```

Or refactor to take advantage of the `compose` feature from `wp.element`:

```js
import { withInstanceId } from '@wordpress/components';
import { compose } from '@wordpress/element';

// ...

const applyConnect = connect(
 	( state ) => {
 		return {
 			excerpt: getEditedPostExcerpt( state ),
 		};
 	},
 	{
 		onUpdateExcerpt( excerpt ) {
 			return editPost( { excerpt } );
  		},
  	}
);

export default compose( [
	applyConnect,
	withInstanceId,
] )( PostExcerpt );
```